src/Certify.Core/Management/PowershellManager.cs

```diff
@@ -197,9 +195,6 @@ public APIResult NewIdentifier(string dns, string alias, string label)
         {
             ps.Commands.Clear();
 
-            // ACME service requires international domain names in ascii mode
-            dns = _idnMapping.GetAscii(dns);
```

Comment:
This method should never receive dns in anything but ASCII. Conversions to ASCII were moved to places where domain names are read from IIS configuration.